common/skills_turn_on_topics_and_patterns.py

```diff
@@ -23,8 +23,21 @@
 from common.travel import TRAVELLING_TEMPLATE, HAVE_YOU_BEEN_TEMPLATE, I_HAVE_BEEN_TEMPLATE
 from common.weather import WEATHER_COMPILED_PATTERN
 from common.bot_persona import YOUR_FAVORITE_COMPILED_PATTERN
+from common.travel_italy import ITALY_PATTERN
 
 SKILL_TRIGGERS = {
+    "dff_travel_italy_skill": {
+        "compiled_patterns": [ITALY_PATTERN],
+        "previous_bot_patterns": [],
+        "detected_topics": [],
+        "intents": [],
+    },
+    "dff_user_kg_skill": {
+        "compiled_patterns": [re.compile(r"(\bpet|pets|hobby|hobbies)")],
```

Comment:
it will also will be activate when `petrovich`, `petsy` words appear

Reply:
@annakorz Take a look please

Reply:
@dilyararimovna, @Ramimashkouk, added word boundary to the pattern